### ISSUE TITLE
Stop making a ton of releases and use the DevOps service hooks directly.

### ISSUE THREAD

**Issue by EraYaN** [jellyfin/jellyfin-docs#303]
And the server should download the artifact same as the windows installer.

@joshuaboniface anything you would need to change?

**Comment by dkanada** [comment]
The documentation doesn't touch the build VM as far as I know, it goes straight to GitHub Pages.

**Comment by EraYaN** [comment]
Well some bot pushes the docs to the github pages, right now using the webhook from github when a release is created. I'd like to remove that step.

**Comment by dkanada** [comment]
We could simply add a build trigger on azure and sidestep the release entirely without needing to change anything.

**Comment by EraYaN** [comment]
Exactly, those service hooks can just fire for the docs pipeline or for every pipeline (and pull in all those artifacts as well instead of having scp credentials on Azure).
